app/build.gradle

```
@@ -1,22 +1,23 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    namespace 'org.literacyapp.familiar_word_reading'
+
+    compileSdk = 35
 
     defaultConfig {
         applicationId "org.literacyapp.familiar_word_reading"
-        minSdkVersion 21
-        targetSdkVersion 23
+        minSdk= 26
+        targetSdk = 35
         versionCode 1000000
         versionName "1.0.0"
```

Bump major version, and add missing `-SNAPSHOT`:
```suggestion
        versionCode 2000000
        versionName "2.0.0-SNAPSHOT"
```